evadb/executor/create_function_executor.py

```diff
@@ -163,6 +164,16 @@ def handle_sklearn_function(self):
             self.node.metadata,
         )
 
+    def convert_to_numeric(self, x):
+        x = re.sub("[^0-9.]", "", str(x))
```

Comment:
This converts arbitrary string into numbers, which can hide real errors from user.

Also, notice the meaning `123,456` is affect by the localization.  Check https://stackoverflow.com/questions/6633523/how-can-i-convert-a-string-with-dot-and-comma-into-a-float-in-python for details. 

My suggestion is:

a) Do nothing. Just throw the error message, forecast expects a numeric column but gets a x type column. To be more advanced, we can suggest some data cleaning method. 
b) Only handling the case that locale supports. For the rest, fallback to throwing error message.

Reply:
Thanks for pointing this out! The issue here is any a times there are dangling strings that might needed to be removed automatically. Like units or currency symbols. I'll look into fixing this via method b.